src/validators/address.js

```diff
@@ -0,0 +1,21 @@
+const regexp = /(?!^([^a-zA-Z]|[\d\s])+$)^.+$/;
+
+/**
+ * Check the validity of supplied address
+ *   - Reject if only consist of numbers, special and whitespace characters
+ *   - Reject if line terminator exist
```

Comment:
I don't think newlines should be rejected. For example, take a look at cermati.com footer, try copy and paste the address there, you'll notice there's newline pasted. Open for discussion

Reply:
Right, when I code this I assume the sender should sanitize the data first, but in the end the consumer must sanitize the data too before use
I'll omit the newline validation